includes/CAPx/Drupal/Importer/Orphans/Lookups/LookupWorkgroupOrphans.php

```diff
@@ -42,10 +42,18 @@ public function execute($orphaner) {
     // workgroup directly. Unfortunately, that means that we need to make
     // another request to the server for each work group.
 
-    $client->setLimit(99999);
+    // Setting limit of items per call to 100 so we don't overload the service.
+    $client->setLimit(100);
```

Comment:
@dscl, This looks good on my first pass. I will have to run it through to test it. 

One comment on this line, can you replace the 100 with the batch limit variable: `variable_get('stanford_capx_batch_limit', 100);`

Thanks